content/applications/finance/accounting/reporting/data_inalterability.rst

```diff
@@ -24,27 +24,72 @@ would break the hash chain.
+<data-inalterability/wizard>`.
+
+- Two indicators are added to the journal entry form view.
+  They show whether the entry is secured or not.
+
+  - A lock icon next to the "Posted" state.
+  - A "Secured" checkbox in the "Other info" tab.
+
+- On journal entry list views a "Not Secured" filter is shown.
+  It can be used to find journal entries that are posted but not secured yet.
+- The menu to open the :ref:`Secure Entries Wizard <data-inalterability/wizard>` is visible.
+
+
+.. _data-inalterability/restricted:
+
+Secure Posted Entries with Hash
+===============================
 
 To start using the hashing function, go to :menuselection:`Accounting --> Configuration > Journals`.
 Open the journal for which you want to activate the feature, go to the :guilabel:`Advanced Settings`
-tab, and enable :guilabel:`Lock Sent Invoices with Hash` for sale journals, or
-:guilabel:`Lock Entries Manually` for purchase and general journals.
-
-For sale journals, the entry is automatically locked once it is sent. You can also click on the
-button :guilabel:`Lock` on the invoice to lock it without sending it. For sale, purchase, and
-general journals, the locking is always done manually using the same button.
+tab, and enable :guilabel:`Secure Posted Entries with Hash`. We call journals for which the
+feature is activated "restricted".
+This feature is available for sales, purchase, and miscellaneous journals.
 
 To compute the hash of an entry, Odoo retrieves the predecessor entries of the chain (i.e., the
 entries with the same sequence prefix) and hashes them in a continuous way from the last hashed
 entry to the new entry to hash.
 
 .. warning::
-   Once you lock an entry in a locked journal, you cannot disable the feature anymore, nor edit any
-   locked entry.
+   Once you post an entry in a restricted journal, you cannot disable the feature anymore, nor edit
+   any secured entry.
+
+.. _data-inalterability/wizard:
+
+Secure Entries Wizard
+=====================
+
+You can also use the :guilabel:`Secure Entries` Wizard to secure journal entries,
+in all journals, up to a specific date.
+
+.. note::
+   The wizard operates independently of the journal settings and journal types.
+   It secures all entries in all journals.
+
+To open it, activate the :ref:`developer mode <developer-mode>`, go to :menuselection:`Accounting
+--> Accounting` and click on :guilabel:`Secure Entries`. In case the :ref:`Inalterability Features
+<data-inalterability/inalterability_features>` are activated it is also visible outside the debug
+mode.
+
+To secure entries, select a date up to which all entries should be secured and press
+:guilabel:`Secure Entries`.
+
+.. warning::
+   After securing the entries you will not be able to edit them any more.
+
+.. note::
+   It can happen that entries are secured that are past the selected date.
+   This is possible since the hash chain corresponds to the sequence prefix,
+   ordered by sequence number.
```

Comment:
```suggestion
   It can happen that entries that are past the selected date are secured.
   This is possible since the hash chain corresponds to the sequence prefix,
   ordered by sequence number.
```